import sys

def error_result():
	return sys.exc_info()[1]